test/models/concerns/host_managed_extensions_test.rb

```diff
@@ -80,14 +78,14 @@ def test_update_does_not_update_content_host
     end
 
     def test_update_with_cv_env
-      host = FactoryGirl.create(:host, :with_content, :content_view => @library_view, :lifecycle_environment => @library)
+      host = FactoryGirl.create(:host)
       host.content_view = @library_view
       host.lifecycle_environment = @library
       assert host.save!
     end
 
     def test_update_with_invalid_cv_env_combo
-      host = FactoryGirl.create(:host, :with_content, :content_view => @library_view, :lifecycle_environment => @library)
+      host = FactoryGirl.create(:host)
```

Comment:
@jlsherrill I updated to just create the host and then add the content view and lifecycle environment